nemo_rl/models/policy/megatron_policy_worker.py

```diff
@@ -658,6 +658,21 @@ def __init__(
             "https://github.com/NVIDIA-NeMo/RL/blob/bccbc377705a81a1f4b3c31ad9767bcc15f735a8/nemo_rl/algorithms/sft.py#L175-L179."
         )
 
+        ## These settings are required for correct gradient computations in mcore
+        ## when calculate_per_token_loss is True, there is no scaling of the gradient in mcore,
+        ## so we handle the scaling in nemo-rl.
+        ## perform_initialization = True is a workaround to ensure the correct tensor parallel attributes are set
+        ## on the TP-sharded parameters.
+        model_cfg.calculate_per_token_loss = True
+        model_cfg.perform_initialization = False
+
+        assert (
+            "aux_loss" not in self.model_cfg.moe_router_load_balancing_type
```

Comment:
`MegatronPolicyWorker` doesn't have a  `model_cfg` member, `self.model_cfg` should be `model_cfg`